x-pack/plugins/apm/server/routes/metrics/by_agent/shared/memory/index.ts

```diff
@@ -63,23 +63,27 @@ export const percentSystemMemoryUsedScript = {
 export const percentCgroupMemoryUsedScript = {
   lang: 'painless',
   source: `
-    /*
-      When no limit is specified in the container, docker allows the app as much memory / swap memory as it wants.
-      This number represents the max possible value for the limit field.
-    */
-    double CGROUP_LIMIT_MAX_VALUE = 9223372036854771712L;
+    if(doc.containsKey('${METRIC_SYSTEM_TOTAL_MEMORY}') && doc.containsKey('${METRIC_CGROUP_MEMORY_USAGE_BYTES}')){
```

Comment:
I'm not sure if this is the way to go about it... will this not unnecessarily return null in some cases? In all honesty it's a little hard to wrap my head around.